lib/tasks/recipient.rake

```diff
@@ -5,4 +5,11 @@ namespace :recipient do
     puts "Enqueing alert for #{subscribers.size} subscribers..."
     subscribers.each { |subscriber| AlertUserJob.perform_later(subscriber.id) }
   end
+
+  desc "Alerting subscribers (async)"
+  task alert_ramadan_subscribers: :environment do
+    subscribers = Recipient.where(subscribed: true, preference: "ramadan")
+    puts "Enqueing alert for #{subscribers.size} subscribers..."
```

Comment:
Rails.logger plutôt http://guides.rubyonrails.org/debugging_rails_applications.html#what-is-the-logger-questionmark